package.json

```diff
@@ -109,29 +111,27 @@
     "@quarterto/chai-dom-equal": "^1.6.0",
     "@quarterto/chai-selector": "^1.3.0",
     "@quarterto/emphasize-monokai-sheet": "^1.3.0",
-    "@quarterto/gaussian": "^1.0.0",
+    "@quarterto/gaussian": "^1.4.0",
     "@quarterto/gaussian-percentile": "^1.0.0",
     "@quarterto/heroku-config-to-env": "^1.3.1",
     "@quarterto/mocha-describe-folder": "^1.3.0",
-    "@quarterto/symbol-logger": "^1.6.0",
-    "amphtml-validator": "^1.0.10",
+    "@quarterto/symbol-logger": "^1.16.0",
```

Comment:
again, absolutely no guarantees this is backwards compatible. it's only used in a [utility script](https://github.com/Financial-Times/google-amp/blob/e62a88e01995e607c5724e4e3461f1447b2bfe3c/scripts/dev-utils/selector-example.js#L13)

Reply:
Seems to work 👌